search/geocoder.hpp

```diff
@@ -101,6 +101,15 @@ class Geocoder
   void GoEverywhere();
   void GoInViewport();
 
+  // Ends geocoding and informs the following stages of the pipeline
+  // (Preranker, Ranker and Emitter).
```

Comment:
Please, omit Ranker and Emitter. Geocoder should know about PreRanker only.
s/Preranker/PreRanker/

Reply:
done